airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -909,6 +916,63 @@ def transform(self, record: MutableMapping[str, Any], stream_slice: Mapping[str,
         return record
 
 
+class ProjectsV2(SemiIncrementalMixin, GithubStream):
+    """
+    API docs: https://docs.github.com/en/graphql/reference/objects#pullrequest
+    """
+
+    is_sorted = "asc"
+    http_method = "POST"
+
+    def path(
+        self, *, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> str:
+        return "graphql"
+
+    def raise_error_from_response(self, response_json):
+        if "errors" in response_json:
+            raise Exception(str(response_json["errors"]))
```

Comment:
should this be `AirbyteTracedException`?

Reply:
removed; replaced with standard `should_retry`